src/commonMain/kotlin/net/kyori/adventure/webui/websocket/Packet.kt

```diff
@@ -0,0 +1,34 @@
+package net.kyori.adventure.webui.websocket
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonObject
+
+/** The client -> server call. */
+public sealed interface Packet
+
+public sealed interface Templates : Packet {
+    public val stringTemplates: Map<String, String>?
+    public val componentTemplates: Map<String, JsonObject>?
+    public val miniMessageTemplates: Map<String, String>?
+}
+
+@Serializable
+@SerialName("call")
+public data class Call(public val miniMessage: String? = null) : Packet
```

Comment:
Mainly a question directed at past kezz, why is this nullable?

Reply:
No clue but i've made like, every field nullable, not sure if that's sane at all